### PR TITLE
Table format support

### PR DESCRIPTION
**Pull request by rex-tension**
Adds an option to process druid sql query results in table format.

### CODE REVIEW

File: src/partials/query.editor.html [hunk 1]
```diff
@@ -10,10 +10,6 @@
         <select class="gf-form-input gf-size-auto" ng-model="ctrl.target.resultFormat" ng-options="f.value as f.text for f in ctrl.resultFormats" ng-change="ctrl.refresh()"></select>
       </div>
     </div>
-    <div class="gf-form max-width-25"  ng-hide="ctrl.target.resultFormat === 'table'">
-      <label class="gf-form-label query-keyword">ALIAS BY</label>
-      <input type="text" class="gf-form-input" ng-model="ctrl.target.alias" spellcheck='false' placeholder="Naming pattern" ng-blur="ctrl.refresh()">
-    </div>
```
Comment (dkhwangbo) [inline src/partials/query.editor.html:16]: Why do you need this change?

File: src/partials/query.editor.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <query-editor-row query-ctrl="ctrl" can-collapse="false" has-text-edit-mode="true">
 <div ng-if="ctrl.target.rawQuery">
  <div class="gf-form">
-    <textarea rows="3" class="gf-form-input" ng-model="ctrl.target.query" spellcheck="false" placeholder="Druid SQL Query" ng-model-onblur ng-change="ctrl.refresh()"></textarea>
+    <textarea rows="20" class="gf-form-input" ng-model="ctrl.target.query" spellcheck="false" placeholder="Druid SQL Query" ng-model-onblur ng-change="ctrl.refresh()"></textarea>
```
Comment (dkhwangbo) [inline src/partials/query.editor.html:4]: Why do you need this change?

File: src/datasource.js [hunk 3]
```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```
Comment (dkhwangbo) [inline src/datasource.js]: I think It's unnecessary changes

**Review by dkhwangbo — COMMENTED** [review]
Hi @rex-tension Thanks for your contribution.

File: src/partials/query.editor.html (hunk 2 above)
Comment (rex-tension) [reply src/partials/query.editor.html:4]: In general, when obtaining metrics directly using SQL, a complex query is required.
In this case, the SQL query will be longer, so the user will have to resize the input box each time.
To reduce this inconvenience, I expanded the default input box size.

File: src/partials/query.editor.html (hunk 1 above)
Comment (rex-tension) [reply src/partials/query.editor.html:16]: This has been removed because it is unimplemented, unnecessary, and confusing.

File: src/datasource.js (hunk 3 above)
Comment (rex-tension) [reply src/datasource.js]: It has been modified. aedd96e

File: src/partials/query.editor.html (hunk 2 above)
Comment (dkhwangbo) [reply src/partials/query.editor.html:4]: Got it

File: src/partials/query.editor.html (hunk 1 above)
Comment (dkhwangbo) [reply src/partials/query.editor.html:16]: Got it. I checked it's similar with InfluxDB plugin but not work.

File: dist/datasource.js [hunk 4]
```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```
Comment (dkhwangbo) [inline dist/datasource.js]: You write this change https://github.com/dkhwangbo/druidplugin/pull/6/commits/aedd96e18330867cfafd72aaefdb84799b3ebf28 but not applied. I think you need rebuild

Comment (rex-tension) [reply dist/datasource.js]: I forgot😲. After building I committed again. 924abf1

**Review by dkhwangbo — APPROVED** [review]
